mutt/memory.c

```diff
@@ -31,12 +31,24 @@
 
 #include "config.h"
 #include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include "memory.h"
 #include "exit.h"
 #include "logging2.h"
 
```

Comment:
I know it's an ugly hack, but please can we follow the style guide.
Please can you create a Doxygen header for this function and fix the position of `{`.